lanl/docker-compose/ochami-krakend-ce.yml

```diff
@@ -1,20 +1,43 @@
 version: '3.7'
 
 services:
-  
+  step-ca:
+    hostname: SI.ca
+    container_name: SI.ca
+    image: ghcr.io/openchami/local-ca:acme
+    ports: 
+      - "443:443"
+    healthcheck: 
+      test: ["CMD-SHELL", "step", "ca", "health", "--ca-url", "https://SI.ca", "--root", "$(step path)/certs/root_ca.crt"]
+      interval: 30s
+      timeout: 10s
+      retries: 5
+      start_period: 10s
+    networks:
+      - internal
+    volumes:
+      - ./configs/step-ca:/mnt
   krakend-ce:
-    container_name: krakend-ce
-    image: devopsfaith/krakend
+    hostname: SI.krakend
+    container_name: SI.krakend
+    image: ghcr.io/openchami/krakend:acme
     environment:
       - KRAKEND_CONFIG=/etc/krakend/krakend.json
       - KRAKEND_PORT=8080
-    hostname: krakend-ce
+    healthcheck:
+      test: ["CMD-SHELL", "CURL_CA_BUNDLE=/root/.acme.sh/SI.krakend_ecc/SI.krakend.cer", "curl", "-f", "https:/SI.krakend:8080/__health"]
+      interval: 30s
+      timeout: 10s
+      retries: 5
+      start_period: 10s
     depends_on:
       - bss
       - smd
+      - step-ca
     ports:
       - "8080:8080"
     networks:
       - internal
```

Comment:
I thought we were planning to separate the internal and external networks involved here so that Krakend-ce could be the only container accessible from outside the docker network.   Should that be part of this PR, or do we expect that in another PR?

Reply:
I think we want that to be part of another PR, at least for the CA I like it being externally accessible so we can have clients bootstrap the core certs.